tensorboard/webapp/metrics/views/right_pane/settings_view_container.ts

```diff
@@ -221,6 +228,14 @@ export class SettingsViewContainer {
     );
   }
 
+  onStepSelectorRangeToggled() {
+    this.store.dispatch(
+      stepSelectorRangeToggled({
+        affordance: TimeSelectionToggleAffordance.CHECK_BOX,
```

Comment:
Would it be worth discssing with Jie-Wei whether it is ok to reuse the TimeSelectionToggleAffordance enum for range toggling or if we should be adding a new enum for Range toggling? It seems like we were adding different enums for different settings and its not clear to me that TimeSelectionToggleAffordance applies to range toggling.

Another possibility, if range can only be toggled on/off one way, is to not specific affordance at all.